colors/sonokai.vim

```diff
@@ -1658,6 +1658,23 @@ highlight! link mkdDelimiter Grey
 highlight! link mkdId Yellow
 " }}}
 " syn_end }}}
```

Comment:
The block should come before `syn_end`, otherwise those highlights won't be included when the `better_performance` option is enabled.